cli/command/cli.go

```diff
@@ -125,25 +133,36 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions) error {
 	if err != nil {
 		return err
 	}
+	cli.clientInfo = ClientInfo{
+		HasExperimental: isExperimentalCli(cli.configFile),
+	}
 	cli.initializeFromClient()
 	return nil
 }
 
+func isExperimentalCli(configfile *configfile.ConfigFile) bool {
+	if e := os.Getenv("DOCKER_EXPERIMENTAL_CLI"); e != "" {
```

Comment:
I don't think this environment variable is useful. Contrary to the DOCKER_ORCHESTRATOR one which you can use to switch between orchestrators, I think switching to experimental features should be a "long term" choice from the user.